app/screens/sso/sso.test.tsx

```diff
@@ -18,31 +20,37 @@ describe('SSO', () => {
     };
 
     test('implement with webview when version is less than 5.32 version', async () => {
-        const store = await configureStore({
-            entities: {
-                general: {
-                    config: {
-                        Version: '5.30.0',
+        const mockStore = configureMockStore();
```

Comment:
Had to fox this because TSC was throwing errors during commit.

Reply:
try again